plugins/account_history_plugin/include/eos/account_history_plugin/account_history_object.hpp

```diff
@@ -13,15 +26,18 @@ class transaction_history_object : public chainbase::object<chain::transaction_h
    id_type             id;
    block_id_type       block_id;
    transaction_id_type transaction_id;
+   AccountName         account_name;
```

Comment:
I'm pretty sure normalizing this will greatly simplify your lookup logic as well.